web/themes/custom/my_awesome_theme/components/03-organisms/site/site-footer/_site-footer.scss

```diff
@@ -14,7 +14,7 @@
 }
 
 .footer:before {
-  background-image: url('https://demo.tagdiv.com/newspaper_pro/wp-content/uploads/2021/06/footer-bg1.jpg');
+  background-image: url('../../../../assets/images/footer-bg1.jpg');
```

Comment:
Have you checked if this is working?
It seems like it wouldn't
If I'm wrong let me know.

btw "Completing requested changes" isn't good commit message, it doesn't describe what changes are in the commit

Reply:
Yes, it is working. Already using this image in the footer.

Reply:
No it isn't